src/main.js

```diff
@@ -20,5 +20,15 @@ var currentCover;
 
 // We've provided one function to get you started
 function getRandomIndex(array) {
-  return Math.floor(Math.random() * array.length);
+  var i = Math.floor(Math.random() * array.length);
+  return array[i]
 }
+
+title.innerText = getRandomIndex(titles);
+tagline1.innerText = getRandomIndex(descriptors);
+tagline2.innerText = getRandomIndex(descriptors);
+  if (tagline1 === tagline2) {
+  tagline2.innerText = getRandomIndex(descriptors);
+  };
+  //come back to this if statement
```

Comment:
We may want to refactor this later.  We added this if statement because there is a very small chance we would get the same descriptor value two times. Still an extremely small chance we could get the same value a third time.